src/MvvmNano.Forms/MvvmNanoFormsPresenter.cs

```diff
@@ -52,9 +51,12 @@ public Page CurrentPage
 
                 Page currentPage = getCurrentPage();
                 var tabbedPage = currentPage as TabbedPage;
```

Comment:
If we check for a NanoMasterDetailPage here, we don't need to have tabbedPage in line 53 before, but after the return statement below.